existing-communities/README.md

```diff
@@ -1 +1,64 @@
 # Existing Communities
+
+## What techniques are currently used to involve a community with software development?
+There are many projects that are build with the help of an online community. The largest projects include the development of the Linux kernel[^linuxrepo], the Apache project[^apacherepo] and the Mozilla browser[^mozillarepo]. To include the online communities during the development of such online software it is necessary to keep everyone up to date and informed about the current state of the project. In short, the means needed can be divided into five categories: communication, version control, issue tracking, testing and package management `citation? wiki: https://en.wikipedia.org/wiki/Open-source_software_development`. For each of these categories will be invested what techniques are currently used by software development projects.
```

Comment:
`citation? wiki: https://en.wikipedia.org/wiki/Open-source_software_development`

This doesn't seem to be working?